lib/matplotlib/axes/_axes.py

```diff
@@ -6147,13 +6147,6 @@ def _normalize_input(inp, ename='input'):
         if bins is None:
             bins = rcParams['hist.bins']
 
-        # xrange becomes range after 2to3
-        bin_range = range
```

Comment:
I have a slight preference for

```python
bin_range = range
del range
```

It causes less code churn and keeps the code below clearer